### PR TITLE
Release 0.11.0

### PR DESCRIPTION
**Pull request by aljinovic**
### Added

- Add official compatibility with Tartiflette 1.4.x. (Pull #153)

### Changed

- Update dependency on Starlette to `0.16.*`. (Pull #154)
- Refactor internal subscription streams (Pull #156)
- Update httpx requirement to `0.19.*` (Pull #159)

### CODE REVIEW

**Comment by aljinovic** [comment]
@florimondmanca could you review please?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
...
+## 0.11.0 - 2021-08-27
+
+### Added
+
+- Add official compatibility with Tartiflette 1.4.x. (Pull #153)
+
+### Changed
+
+- Update dependency on Starlette to `0.16.*`. (Pull #154)
+- Refactor internal subscription streams (Pull #156)
```
Comment (florimondmanca) [inline CHANGELOG.md:17]: Don't think we should include this in the changelog, as it's a purely internal, non-user-impacting change?

```suggestion
```

**Review by florimondmanca — APPROVED** [review]
Thank you, @aljinovic!

I'll go ahead and commit the suggestion I've just done here, since I think it's non-controversial, then merge and release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.11.0 - 2021-08-27
```
Comment (florimondmanca) [inline CHANGELOG.md:8]:
```suggestion
## 0.11.0 - 2021-09-03
```

**Comment by florimondmanca** [comment]
@aljinovic I'm having issues logging into PyPI to perform the release due to lost 2FA. I've put up an issue on PyPI to get help, but the release should be delayed a tad again.

**Pull request merged**